suave/sol/libraries/Suave.sol

```diff
@@ -150,13 +203,20 @@ library Suave {
         return data;
     }
 
+    /// @notice Handles the storage of data in the confidential store. Requires the caller to be part of the `AllowedPeekers` for the associated bid.
```

Comment:
"Handles the storage" is unclear -- would "Stores data in the confidential store?" be clearer?